scripts/migrate-live-blue-green-step-3.1.bash

```diff
@@ -16,10 +16,11 @@ function git_current_branch() {
   fi
   echo ${ref#refs/heads/}
 }
+[[ $(uname) -eq "Darwin" ]] && NL=$'\\\n' || NL="\n"
 
 for file in $(cat "$APPLICATION_FILE"); do
   cd $file
-  sed -Ei 's/host: (.*)/host: \1\n    contextColour: green/' helm_deploy/values-${ENV_SUFFIX}.yaml
+  sed -Ei '' 's/host: (.*)/host: \1$NL    contextColour: green/' helm_deploy/values-${ENV_SUFFIX}.yaml
```

Comment:
on a mac `sed` is BSD sed instead so have to specify the backup extension explicitly